dependencies.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "pypi-dependencies",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "python3-PyGObject",
```

Comment:
PyGObject is in the GNOME runtime, isn't it?

Reply:
removed dependencies and it still builds... you're right :)

Reply:
yes ^^